recipes/zga/meta.yaml

```diff
@@ -36,6 +36,7 @@ requirements:
     - dfast
     - nxtrim
     - racon
+    - minimap2
```

Comment:
you need to bump the build number as well

Reply:
> you need to bump the build number as well
Dear Björn,
should i make new PR with this change?

Reply:
No, you can change this PR.

Reply:
> No, you can change this PR.

I did it, but now I got an error:
ERROR: recipes/zga/meta.yaml:14: build_number_needs_reset: The recipe build number should be reset to 0